<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AudienceWidget</class>
 <widget class="QWidget" name="AudienceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="5,3">
   <item>
    <widget class="QTableWidget" name="tableWidget"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Фильтр</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="numberLabel">
          <property name="text">
           <string>Номер</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="numberComboBox"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="buildingLabel">
          <property name="text">
           <string>Корпус</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="buildingComboBox"/>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="filterPushButton">
          <property name="text">
           <string>Применить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
